src/Timer.cc

```diff
@@ -52,15 +59,18 @@ bool Timer::Running() const
 }
 
 //////////////////////////////////////////////////
-Time Timer::Elapsed() const
+double Timer::Elapsed() const
```

Comment:
If `Timer` will be deprecated, we shouldn't bother to change this function.

If `Timer` won't be deprecated, I'd suggest returning a chrono type here. And we should deprecate this function before removing it. Maybe the new function can be called `ElapsedTime` or something similar.